File: muridku/muridku.backend/GlobalDbProcessor/Muridku.QueryRequestReceiver/Models/Dbs/KtbHistory.cs
﻿using System;

namespace Muridku.QueryRequestReceiver.Models.Dbs
{
  public class KtbHistory : ModelDbBase
  {
    public long ktb_id { get; set; }
    public DateTime meet_dt { get; set; }
    public long material_id { get; set; }
    public string material_name { get; set; }
    public int material_chapter { get; set; }
    public int is_active { get; set; }
  }
}

File: muridku/muridku.web/app/Models/KtbHistory.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Notifications\Notifiable;

class KtbHistory extends Model
{
    use HasFactory, Notifiable;

    protected $table = 'ktbhistory';
    const CREATED_AT = 'dtm_crt';
    const UPDATED_AT = 'dtm_upd';

    protected $fillable = [
        'meet_dt',
        'material_name',
        'material_chapter',
    ];

    protected $hidden = [
        'ktb_id',
        'material_id',
        'is_active',
        'dtm_crt',
        'usr_crt',
        'dtm_upd',
        'usr_upd'
    ];
}
